.env.development

```diff
@@ -12,8 +13,9 @@ REACT_APP_ETHER_SCAN_URL=https://goerli.etherscan.io/address/
 REACT_APP_BLOCKNATIVE_KEY=3bac81d8-e68e-406c-a1c4-889862ff18f0
 REACT_APP_COIN_EXCHANGE_KEY=57B4AD6E-A85C-4B43-ADD0-FF4BBB3600F8
 REACT_APP_GOOGLE_TAG_URL=http://www.googletagmanager.com/gtm.js?id=
-REACT_APP_SSV_API_ENDPOINT=https://api.stage.ssv.network/api/v3
-REACT_APP_SSV_CONTRACT_ADDRESS=0x6471F70b932390f527c6403773D082A0Db8e8A9F
+#REACT_APP_SSV_API_ENDPOINT=https://api.stage.ssv.network/api/v3
+#REACT_APP_SSV_CONTRACT_ADDRESS=0x6471F70b932390f527c6403773D082A0Db8e8A9F
+#REACT_APP_SSV_CONTRACT_ADDRESS_V4=0x6471F70b932390f527c6403773D082A0Db8e8A9F
```

Comment:
If you don't use this anymore - remove instead of commenting

Reply:
done